addons/binding/org.openhab.binding.sleepiq/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,140 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="sleepiq"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+    <!-- SleepIQ Cloud Service -->
+    <bridge-type id="cloud">
+        <label>SleepIQ Cloud</label>
+        <description>The SleepIQ cloud service provides connections to all beds in an account.</description>
+
+        <properties>
+            <property name="vendor">Select Comfort</property>
+        </properties>
+        
+        <config-description>
+            <parameter name="username" type="text">
+                <label>Username</label>
+                <description>Username of a registered SleepIQ account owner</description>
+                <required>true</required>
+            </parameter>
+            <parameter name="password" type="text">
+                <context>password</context>
+                <label>Password</label>
+                <description>Password of a registered SleepIQ account owner</description>
```

Comment:
As you noted, the SleepIQ API doesn't support keys. If it does in the future, I will remove the use of user credentials.